code/.eslintrc.json

```diff
@@ -41,6 +41,10 @@
       {
         "allowSingleLine": true
       }
+    ],
```

Comment:
I also do this in all projects due to working with a Windows-computer = timesaver :D